accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservice/AceService.java

```diff
@@ -62,13 +70,13 @@
 
     /** Common entry point for JMX and install hook.
      * 
-     * @param session
      * @param history
      * @param configurationFileContentsByFilename
      * @param authorizableInstallationHistorySet
+     * @param restrictedToPaths only apply ACLs to root paths as given
      * @throws Exception */
-    public void installConfigurationFiles(Session session, AcInstallationHistoryPojo history,
```

Comment:
I still want to be able to give a dedicated session to the AcService, why has that been removed? Even with intermediate saves there should only be one session used IMHO (and that one should be given as parameter).